devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -73,12 +73,9 @@ def parseAttrMessage(description) {
 		buttonNumber = Integer.valueOf(descMap?.command[1].toInteger()) + 1
        
 		log.debug "Number is ${buttonNumber}"
-		def event = createEvent(name: "button", value: "pushed", data: [buttonNumber: buttonNumber], descriptionText: "pushed", isStateChange: true)
-		if (buttonNumber != 1) {
-			sendEventToChild(buttonNumber, event)
-		} else {
-			sendEvent(event)
-		}
+		def descriptionText = (device.displayName.endsWith(' 1') ? "${device.displayName[0..-2]} button" : "${device.displayName}") + " ${buttonNumber} was pushed"
```

Comment:
This code is very similar to the code that calculates the `componentLabel` in `addChildButtons`. Please make a function that both of these usages can share so that the coupling is clear.